crates/ibc/src/events.rs

```diff
@@ -137,12 +135,12 @@ pub enum IbcEventType {
     OpenConfirmChannel,
     CloseInitChannel,
     CloseConfirmChannel,
+    ChannelClosed,
```

Comment:
I actually like that it's inconsistent with the handshake events, since it isn't one. Plus it's the same name as ibc-go, making the correspondence between the 2 implementations easier. WDYT?